gr-qtgui/python/qtgui/qa_qtgui.py

```diff
@@ -78,6 +78,11 @@ def test13(self):
     def test14(self):
         self.qtsnk = qtgui.eye_sink_c(1024, 1, 1, None)
 
+    def test15(self):
+        self.qtsnk = qtgui.matrix_sink("Dopler", 2, 4, False,
```

Comment:
Doppler is written with two "p", because the guy who discovered / described the Doppler effect was called Doppler.  :grin:

Reply:
oops , I'll correct that